frontend/vue/components/Statevector/StatevectorBaseWidget.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="statevector-base-widget">
+    <div class="statevector-base-widget__explanation">
+      <slot />
```

Comment:
should we wrap this in a `<p>` for semantics?

Reply:
Good question. It could be a good option, but I don't know if it makes sense in any case.